components/list/columns/OrganizationType.vue

```diff
@@ -5,14 +5,18 @@
 </template>
 
 <script>
+import { mapGetters } from 'vuex';
 
 export default {
-  components: { },
-  props     : { organizationTypeId: { type: String, required: true } },
-  methods   : { getOrganizationNameById },
+  props   : { organizationTypeId: { type: String, required: true } },
+  computed: {
+    ...mapGetters({ getTypeById: 'organizations/getTypeById' }),
+  },
+  methods: { getOrganizationNameById },
 };
 
 function getOrganizationNameById(organizationTypeId) {
-  return this.$store.getters['organizations/typeNameById'](organizationTypeId);
+  const organization = this.getTypeById(organizationTypeId);
+  return organization ? organization.Title : '';
```

Comment:
if type not found return the ID (help in debug)
also the variable name is wrong. should be `type` or `organizationType`
```suggestion
  const organizationType = this.getTypeById(organizationTypeId);
  return organizationType ? organizationType.Title : organizationTypeId;
```

Reply:
Done